graalwasm/graalwasm-micronaut-excelice/src/main/java/com/example/ExcelizeService.java

```diff
@@ -0,0 +1,258 @@
+package com.example;
+
+import io.micronaut.core.io.ResourceResolver;
+import io.micronaut.core.io.scan.ClassPathResourceLoader;
+import org.graalvm.polyglot.*;
+import org.graalvm.polyglot.io.IOAccess;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+@io.micronaut.context.annotation.Context
+public class ExcelizeService {
+
+    private final ResourceResolver resourceResolver;
+
+    public ExcelizeService(ResourceResolver resourceResolver) {
+        this.resourceResolver = resourceResolver;
+    }
+
+
+    public void runExcelizeComplete(Object[][] array) throws IOException {
+        // Constants
+        final String INTERNAL_MODULE_URI_HEADER = "oracle:/mle/";
+
+        // Helper method for reading files
+        byte[] excelizeWasmBytes  = resourceResolver.getResourceAsStream("classpath:excelize.wasm").get().readAllBytes();
+
+        // Load test file based on fileSize
+        String test = null;
+        System.out.println("Executing excelize ");
+
+        test = Files.readString(Paths.get("./src/main/resources/excelize_test.js"));
+        // Load required JavaScript files
+
+        String prep = Files.readString(Paths.get("./src/main/resources/excelize_prep.js"));
+        String encodingIdxs = Files.readString(Paths.get("./src/main/resources/encoding-indexes.js"));
+        String encoding = Files.readString(Paths.get("./src/main/resources/encoding.js"));
+        String excelizeLib = Files.readString(Paths.get("./src/main/resources/excelize_m.js"));
```

Comment:
1. Please use resourceResolver for all wasm and js files.
2. Creating a new engine and context per request is not ideal. GraalJS and GraalWasm support engine sharing, which would be better. For good throughput, it would makes sense to [pool contexts like it's done in the Photon demo](https://github.com/graalvm/graal-languages-demos/blob/0951909cb64a04262eaa6a27bc5800846e785019/graalwasm/graalwasm-micronaut-photon/src/main/java/com/example/PhotonPool.java). The idea is to run all the initialization code only once when the contexts are created and then every request will use a context from the pool to handle the request.